### ISSUE TITLE
Can't run when using neural-tangents 0.3.2

### ISSUE THREAD

**Issue by xiaulinhu** [tancik/fourier-feature-networks#5]
When I use  neural-tangents 0.3.2 to run the 1d_regression.ipynb, I got ValuError:
![image](https://user-images.githubusercontent.com/56224052/91798557-bd45ce80-ec57-11ea-9408-2c10e2230f09.png)

Could you help me with this?

**Comment by tancik** [comment]
Yep, it looks like the neural_tangents 0.3.2 has some breaking changes. I updated the notebook to use version 0.2.2.

**Comment by fengzlj** [comment]
Hello, When I use neural-tangents 0.2.2 to run the 1d_regression.ipynb, I got AttributeError: 'ShapedArray' object has no attribute 'val':
![image](https://user-images.githubusercontent.com/46813334/107144244-eb50dc80-6974-11eb-8dd1-73af22b0914a.png)
Could you help me with this?

**Comment by chenshine1** [comment]
> Hello, When I use neural-tangents 0.2.2 to run the 1d_regression.ipynb, I got AttributeError: 'ShapedArray' object has no attribute 'val':
> ![image](https://user-images.githubusercontent.com/46813334/107144244-eb50dc80-6974-11eb-8dd1-73af22b0914a.png)
> Could you help me with this?

you should try to use jax==0.1.76
